docker-compose-dev.yml

```diff
@@ -94,6 +94,30 @@ services:
       - DATABASE_PASSWORD=testkeycloakpassword
     networks:
       - safe-tier
+    volumes:
+      - program-artifacts:/usr/src/app/media/
+    depends_on:
+      - keycloak
+      - postgres
+  gateway_scheduler:
+    container_name: gateway_scheduler
+    build:
+      context: ./
+      dockerfile: infrastructure/docker/Dockerfile-gateway
+    entrypoint: "./scripts/scheduler.sh"
+    environment:
+      - DEBUG=0
```

Comment:
nit: we set DEBUG=1 in the regular gateway, should we do the same here (or set the gateway to DEBUG=0)?